src/installer.ts

```diff
@@ -164,6 +164,22 @@ async function resolveVersionFromManifest(
   }
 }
 
+export async function addExecutablesToCache(
+  extPath: string,
+  info: IGoVersionInfo,
+  arch: string
+): Promise<string> {
+  core.info('Adding to the cache ...');
+  const cachedDir = await tc.cacheDir(
+    extPath,
+    'go',
+    makeSemver(info.resolvedVersion),
+    arch
+  );
+  core.info(`Successfully cached go to ${cachedDir}`);
+  return cachedDir;
+}
+
```

Comment:
I'd return calls of core.info() into the installGoVersion() function and not wrap tc.cacheDir() into additional function as it's pretty small and reusable piece of code by itself.